specification/entities.yaml

```diff
@@ -478,3 +478,23 @@ components:
       required:
         - schema_url
         - metadata
+
+    DataSource:
+      type: object
+      properties:
+        oddrn:
+          type: string
+        name:
+          type: string
+        description:
+          type: string
+        namespace:
```

Comment:
Why should we provide a namespace on ingestion?

Reply:
Removed